backend/src/Controller/Commander/DataCommanderController.php

```diff
@@ -50,29 +52,37 @@ public function appendNewData(KernelInterface $kernel): Response
         return new Response($content);
     }
 
-//    /**
-//     * @Route("reenternewdata", name="reenterNewData", methods={"POST"})
-//     * @IsGranted("ROLE_SUPER_ADMIN")
-//     * @param KernelInterface $kernel
-//     * @return Response
-//     */
-//    public function createNewData(KernelInterface $kernel): Response
-//    {
-//        $application = new Application($kernel);
-//        $application->setAutoExit(false);
-//
-//        $input = new ArrayInput([
-//            'command' => 'hautelook:fixtures:load'
-//        ]);
-//
-//        // You can use NullOutput() if you don't need the output
-//        $output = new BufferedOutput();
-//        $application->run($input, $output);
-//
-//        // return the output, don't use if you used NullOutput()
-//        $content = $output->fetch();
-//
-//        // return new Response(""), if you used NullOutput()
-//        return new Response($content);
-//    }
+    /**
+     * @Route("reenternewdata", name="reenterNewData", methods={"POST"})
+     * @IsGranted("ROLE_SUPER_ADMIN")
+     * @param KernelInterface $kernel
+     * @return Response
+     * @throws Exception
+     */
+    public function createNewData(KernelInterface $kernel): Response
```

Comment:
this is good, but we do not need it at all, 
after drop DB we can call just service for fill DB again.

Reply:
meaning we do not need controller or api for it

Reply:
If we need to use it in the system control app, we need an API, right?

Reply:
no, as I said in my first comment in second line